pyproject.toml

```diff
@@ -53,6 +53,7 @@ python-source = "python"
 module-name = "pydantic_core._pydantic_core"
 bindings = 'pyo3'
 features = ["pyo3/extension-module"]
+profile = "release"  # TEMPORARY: remove this
```

Comment:
```suggestion
```

... just wondering why this was necessary? Maybe our dev docs were missing something.

Reply:
Did the codspeed step there use release mode when pointing to pydantic core branch?

Reply:
Does @sydney-runkle know if Pydantic side CodSpeed bench CI step installs a release version of Pydantic-core when pointing to a Pydantic-core branch? So the benchmarks won't go wonky there if this temp profile hack is removed ^

Reply:
So I am reasonably sure you can remove this. Easy to add it back :)

Reply:
Thanks, it's gone now

Reply:
Codespeed worked fine there in other PR